locale/fr/manageiq.po

```diff
@@ -26895,6 +26901,9 @@ msgstr "Identifiants d'authentification non valides"
 msgid "Invalid button action"
 msgstr "Action de bouton non valide"
 
+msgid "Invalid button action!"
+msgstr "Action de bouton non valide !"
```

Comment:
Space before the `!`?  (doesn't matter cause this should go away, I'm just surprised)

Reply:
fixed anyway